Spatial.gd

```diff
@@ -822,16 +815,14 @@ func _on_NewPathPoint_pressed():
 #
 ################################################################################
 func _on_SavePath_pressed():
-	$Control/Dialogs/SaveDialog.show()
+	#TODO: Save split files into individual files
+	pass
 
 ################################################################################
 # TODO: This function will be used when exporting packs
 ################################################################################
 func _on_SaveDialog_file_selected(path):
-	self.markerdata[str(self.map_id)] = data_from_renderview()
-	var save_game = File.new()
-	save_game.open(path, File.WRITE)
-	save_game.store_string(JSON.print(self.markerdata))
+	pass
```

Comment:
If this function is not doing anything is is needed anymore?

Reply:
You're correct. I'll remove it.